snap/snapcraft.yaml

```diff
@@ -342,57 +349,56 @@ parts:
     plugin: nil
     source: ./src/client/gui
     stage-packages:
-      - libgdk-pixbuf2.0-bin
-      - libglib2.0-0
-      - libgtk-3-0
-      - libkeybinder-3.0-dev
+    - libgdk-pixbuf2.0-bin
+    - libglib2.0-0
+    - libgtk-3-0
+    - libkeybinder-3.0-dev
     build-packages:
-      - libkeybinder-3.0-dev
-      - build-essential
-      - clang
-      - ninja-build
-      - unzip
+    - libkeybinder-3.0-dev
+    - build-essential
+    - clang
+    - ninja-build
+    - unzip
     stage:
-      - -**/X11
-      - -usr/include
-      - -usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libcolordprivate.so.*
-      - -usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libdconf.so.*
-      - -usr/lib/*/libgirepository-1.0.so.*
-      - -usr/lib/*/libgthread-2.0.so*
-      - -usr/lib/*/libicuio.so.*
-      - -usr/lib/*/libicutest.so.*
-      - -usr/lib/*/libicutu.so.*
-      - -usr/share/apport
-      - -usr/share/bug
-      - -usr/share/*doc*
-      - -usr/share/gettext
-      - -usr/share/icons/Adwaita
-      - -usr/share/icons/DMZ-Black
-      - -usr/share/icons/Humanity*
-      - -usr/share/icons/LoginIcons
-      - -usr/share/icons/hicolor
-      - -usr/share/icons/ubuntu-*
-      - -usr/share/libthai
-      - -usr/share/lintian
-      - -usr/share/man
-      - -usr/share/mime
-      - -usr/share/pkgconfig
-      - -usr/share/thumbnailers
+    - -**/X11
+    - -usr/include
+    - -usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libcolordprivate.so.*
+    - -usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libdconf.so.*
+    - -usr/lib/*/libgirepository-1.0.so.*
+    - -usr/lib/*/libgthread-2.0.so*
+    - -usr/lib/*/libicuio.so.*
+    - -usr/lib/*/libicutest.so.*
+    - -usr/lib/*/libicutu.so.*
+    - -usr/share/apport
+    - -usr/share/bug
+    - -usr/share/*doc*
+    - -usr/share/gettext
+    - -usr/share/icons/Adwaita
+    - -usr/share/icons/DMZ-Black
+    - -usr/share/icons/Humanity*
+    - -usr/share/icons/LoginIcons
+    - -usr/share/icons/hicolor
+    - -usr/share/icons/ubuntu-*
+    - -usr/share/libthai
+    - -usr/share/lintian
+    - -usr/share/man
+    - -usr/share/mime
+    - -usr/share/pkgconfig
+    - -usr/share/thumbnailers
```

Comment:
Yeah, it was definitely a drive by change.  Sorry for the noise it caused :slightly_frowning_face: I do know better than to normally do this, but I figured if I didn't do it here, then it probably won't get done.  I can make it it's own commit or get rid of it altogether if you prefer.

Reply:
Hmmm. I think it's a necessary change, because it will make the lists in the file look consistent. So I would vote for sending them to a new commit :)

Reply:
Ok, I will make it it's own commit in this PR so it's clear that is what changed.  Thanks!

Reply:
Ok, it is split out now,  Thanks!